test/paymentmethods.collective.to.collective.test.js

```diff
@@ -0,0 +1,438 @@
+import * as store from './features/support/stores';
+
+const ORDER_TOTAL_AMOUNT = 1000;
+
+const createOrderQuery = `
+  mutation createOrder($order: OrderInputType!) {
+    createOrder(order: $order) {
+      id
+      fromCollective {
+        id
+        slug
+      }
+      collective {
+        id
+        slug
+      }
+      subscription {
+        id
+        amount
+        interval
+        isActive
+        stripeSubscriptionId
+      }
+      totalAmount
+      currency
+      description
+    }
+  }
+`;
+
+
+describe("payments.collectiveToCollective.test.js", () => {
+  const STRIPE_FEE_STUBBED_VALUE = 300;
+  let sandbox, user1, user2, transactions,  collective1, collective2, collective3, host1, host2,
+      organization, stripePaymentMethod, openCollectivePaymentMethod;
+
+  before(async () => {
+    await utils.resetTestDB();
+  });
+
+  describe('validation', () => {
+    it('validates the token for Stripe', (done) => {
+      models.PaymentMethod.create({ service: 'stripe', type: 'creditcard', token: 'invalid token' })
+        .catch(e => {
+          expect(e.message).to.equal("Invalid Stripe token invalid token");
+          done();
+        })
+    })
+  });
+
+  describe('Collective to Collective Transactions', () => {
+
+    before('creates User 1', () => models.User.createUserWithCollective({ name: "User 1" }).then(u => user1 = u));
+    before('creates User 2', () => models.User.createUserWithCollective({ name: "User 2" }).then(u => user2 = u));
+    before('create Host 1', () => models.Collective.create({ name: "Host 1", currency: "USD", isActive: true }).then(c => host1 = c));
+    before('create Host 2', () => models.Collective.create({ name: "Host 2", currency: "USD", isActive: true }).then(c => host2 = c));
+    before('create Collective 1', () => models.Collective.create({ name: "Collective 1", currency: "USD", HostCollectiveId: host1.id, isActive: true }).then(c => collective1 = c));
+    before('create Collective 2', () => models.Collective.create({ name: "Collective 2", currency: "USD", HostCollectiveId: host1.id, isActive: true }).then(c => collective2 = c));
+    before('create Collective 3', () => models.Collective.create({ name: "Collective 3", currency: "USD", HostCollectiveId: host2.id, isActive: true }).then(c => collective3 = c));
+    // before('Add User 1 as Admin of Collective 1', () => collective1.editMembers([{ id: user1.id, MemberCollectiveId: user1.CollectiveId, role: 'ADMIN' }]));
+    before('create an organization', () => models.Collective.create({ name: "pubnub", currency: "USD" }).then(o => organization = o));
+    before('create a payment method', async () => models.PaymentMethod.create({
+      name: '4242',
+      service: 'stripe',
+      type: 'creditcard',
+      token: 'tok_123456781234567812345678',
+      CollectiveId: organization.id,
+      monthlyLimitPerMember: 10000
+    }).then(pm => stripePaymentMethod = pm));
+    beforeEach('create many transactions', async () => {
```

Comment:
I'd be more specific: "create transactions for 3 donations from organization to collective1"